src/components/notifications/notifications.jsx

```diff
@@ -0,0 +1,18 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import styles from './notifications.module.css';
+
```

Comment:
```suggestion

const icons = {
  warning: IconWarning,
  success: IconSuccess,
  error: IconError
};

```
тут можна створити об'єкт з іконками, щоб потім можна було отримати потрібнку в залежності від значення `type`